app/providers/atmosphere/optimization_strategy/default.rb

```diff
@@ -35,17 +35,39 @@ def can_scale_manually?
       def vmt_candidates_for(appliance)
         vmts = VirtualMachineTemplate.where(
           appliance_type: appliance.appliance_type,
-          state: 'active',
+          state: 'active'
         )
-        vmts.select do |vmt|
-          (vmt.tenants & appliance.tenants.active.funded_by(appliance.fund)).present?
+        if appliance.tenants.present?
+          vmts = restrict_by_user_requirements(vmts, appliance)
         end
+        vmts = restrict_by_tenant_availability(vmts, appliance)
+        vmts
       end
 
       private
 
       attr_reader :appliance
 
+      def restrict_by_user_requirements(vmts, appliance)
+        # If the user requests that the appliance be bound to a specific set of tenants,
+        # the optimizer should honor this selection.
+
+        # Workaround
```

Comment:
I'm not sure if I understand this comment. Can you explain why this is workaround?